core/Cargo.toml

```diff
@@ -65,6 +65,9 @@ tracing = { workspace = true, default-features = false }
 # AccountId20
 keccak-hash = { workspace = true}
 
+# Error handling 
+thiserror = { workspace = true, default-features = false }
```

Comment:
I'd just lump this with the main blob of dependencies :)

Reply:
done